Core/include/Acts/EventData/SpacePointProxy2.hpp

```diff
@@ -67,120 +76,117 @@ class SpacePointProxy2 {
   /// @throws std::out_of_range if the index is out of range.
   /// @throws std::logic_error if no source links column is available.
   /// @throws std::logic_error if source links are already assigned to the space point.
-  void assignSourceLinks(std::span<const SourceLink> sourceLinks)
+  void assignSourceLinks(std::span<const SourceLink> sourceLinks) const
     requires(!ReadOnly)
   {
     m_container->assignSourceLinks(m_index, sourceLinks);
   }
 
   /// Mutable access to the source links of the space point.
   /// @return A mutable span of source links associated with the space point.
-  std::span<SourceLink> sourceLinks() noexcept
+  std::span<SourceLink> sourceLinks() const noexcept
     requires(!ReadOnly)
   {
     return m_container->sourceLinks(m_index);
   }
   /// Mutable access to the x coordinate of the space point.
   /// @return A mutable reference to the x coordinate of the space point.
-  float &x() noexcept
+  float &x() const noexcept
     requires(!ReadOnly)
   {
     return m_container->x(m_index);
   }
```

Comment:
I won't ask you to complicate this any further, so feel free to disregard, but this means if you have a const-ref to a mutable proxy, you can mutate the contents, right?